lepidopter-vmdebootstrap_build.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -exa
+GIT_BUILD="$(git describe)"
```

Comment:
Yes. Reading the manual, the problem that `--tags` solves is different. By default `git describe` only considers annotated tags and ignores other tags. So, perhaps it is more robust.

Example:

```
$ pwd
measurement_kit
$ git rev-parse HEAD
f856637a22920e935b71a8b21cd08ce2a6c968e8
$ git describe
fatal: No annotated tags can describe 'f856637a22920e935b71a8b21cd08ce2a6c968e8'.
However, there were unannotated tags: try --tags.
$ git describe --tags
v0.2.0-beta.1-1-gf856637
```

So, since I do not use annotated tags, the bare command does not work for me. If you however always use annotated tags your usage is fine.